docs/zh/latest/plugins/http-logger.md

```diff
@@ -40,6 +40,7 @@ description: 本文介绍了 API 网关 Apache APISIX 的 http-logger 插件。
 | uri                    | string  | 是     |               |                      | HTTP 或 HTTPS 服务器的 URI。                   |
 | auth_header            | string  | 否     |               |                      | 授权 header（如果需要）。                                    |
 | timeout                | integer | 否     | 3             | [1,...]              | 发送请求后保持连接处于活动状态的时间。           |
+| log_format             | object  | 否     |               |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../apisix-variable.md) 或 [NGINX 内置变量](http://nginx.org/en/docs/varindex.html)。 |
```

Comment:
Should we synchronize the word `Nginx` with en docs?

Reply:
This PR has changed the Nginx to NGINX:
https://github.com/apache/apisix/pull/6886/files#diff-642ed4e3cc78101ac869166d3568f8910fdca52ab5483c241c775e6411eaf985

I am not sure which one is preferred by the doc team. But I am sure it is not just about the logger variable.